android/src/main/java/com/rnfingerprint/FingerprintHandler.java

```diff
@@ -0,0 +1,83 @@
+package com.rnfingerprint;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.hardware.fingerprint.FingerprintManager;
+import android.os.CancellationSignal;
+import android.support.v4.app.ActivityCompat;
+import android.widget.Toast;
+
+public class FingerprintHandler extends FingerprintManager.AuthenticationCallback {
+
+    private CancellationSignal cancellationSignal;
+    private boolean selfCancelled;
+    private Context appContext;
+
+    private final FingerprintManager mFingerprintManager;
+    private final Callback mCallback;
+
+    public FingerprintHandler(Context context, FingerprintManager fingerprintManager, Callback callback) {
+        appContext = context;
+        mFingerprintManager = fingerprintManager;
+        mCallback = callback;
+    }
+
+    public boolean isFingerprintAuthAvailable() {
+        return (android.os.Build.VERSION.SDK_INT >= 23)
+                && mFingerprintManager.isHardwareDetected()
+                && mFingerprintManager.hasEnrolledFingerprints()
+                && (ActivityCompat.checkSelfPermission(appContext,
+                        Manifest.permission.USE_FINGERPRINT) ==
+                        PackageManager.PERMISSION_GRANTED);
```

Comment:
Same permission comment as before, also, this function seems a duplicated from `FingerprintAuthModule.java` ?